scripts/vagrant/zeppelin-dev/roles/python-addons/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+# setup zeppelin python dependencies
+# Assumes Ubuntu Server 14.04 LTS (HVM) - ami-f64f77b3    64 bit
+# Assumes Java 7 installed as part of the top level playbook
+---
+
+- name: Install python dependencies. This will take a while.
+  apt: pkg={{ item }} state=latest
+  with_items:
+    - python-pip
+    - python-matplotlib
+    - python-scipy
+    - python-numpy
+    - python-pandas
+
+# Change python backend that pyspark can render 
+# matplotlib plots without an interactive window
+# Fully quoted because of the ': ' on the line.
+- name: use Agg as pythons default window instead of tkAgg
+  lineinfile: "dest=/etc/matplotlibrc regexp='^backend' line='backend : Agg'"
```

Comment:
The ideas is to make the VM dead easy to use with zeppelin and pyspark.  I think you can set the backend to tkAgg again in a python script.  I recall setting the backend to Agg  in the python script did not solve the pyspark rendering issue.  This was the solution stated by users on stackoverflow actually solved it this way.  It is akin to an environment level setting from what I understand about the matplotlib backend.